src/mplex.ts

```diff
@@ -301,7 +297,7 @@ export class MplexStreamMuxer implements StreamMuxer {
         }
 
         // We got data from the remote, push it into our local stream
-        stream.source.push(message.data.subarray())
+        stream.source.push(message.data)
```

Comment:
using `subarray()` is way better than before, however I think we don't even need it since it's already `Uint8Array`

<img width="855" alt="Screen Shot 2022-08-03 at 15 53 04" src="https://user-images.githubusercontent.com/10568965/182567085-e5d1f950-b8fe-45f3-85cf-e3c4c0227844.png">